src/test/compilation-failed/const-eval-failed.spec.ts

```diff
@@ -163,4 +163,9 @@ describe("fail-const-eval", () => {
         errorMessage:
             "Cannot evaluate expression to a constant: repeat argument must be a number between -2^256 (inclusive) and 2^31 - 1 (inclusive)",
     });
+    itShouldNotCompile({
```

Comment:
Let's also add a negative test with `ascii("")`

Reply:
I just checked: in Func `""c` results in `0 PUSHINT`

Reply:
btw, `ascii("⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡")` should result in overflow, but `ascii("⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡")` should be equal `1563963554659859369353828835329962428465513941646011501275668087180532385` (let's add these two tests too)